README.md

```diff
@@ -23,7 +23,20 @@ to change the interface of your classes or to change any other Ruby code.
 
 As simple as Ruby, as efficient as Rust.
 
-## Examples
+## Contents
+
+* [The famous `String#blank?` method](#the-famous-stringblank-method)
+* [Simple Sidekiq-compatible server](#simple-sidekiq-compatible-server)
+* [Safe-conversions](#safe-conversions)
+* [True parallelism](#true-parallelism)
+* [Defining-a-new-class](#defining-a-new-class)
+* [Replacing only several methods instead of the whole class](#replacing-only-several-methods-instead-of-the-whole-class)
+* [Class definition DSL](#class-definition-dsl)
+* [Calling Ruby code from Rust](#calling-ruby-code-from-rust)
```

Comment:
I would say everything above this should be under "Examples", an h2 header right below this list, and the bullet points above should be indented.